src/frontend/components/login/Login.vue

```diff
@@ -30,35 +31,56 @@
                 name="password"
                 :error="!!errors.password"
                 :error-messages="errors.password"
-                prepend-icon="lock"
+                prepend-icon="fingerprint"
                 ref="password"
                 onfocus="this.select()"
                 type="password"></v-text-field>
             </v-card-text>
 
             <v-card-actions>
-              <nuxt-link to="/passwordReset">Forgot password?</nuxt-link>
+              <v-btn type="submit" style="color: green;" v-on:click="guestHome"  
+                to="/guestHome">
+                <strong>guest</strong>
+              </v-btn>
+              <v-btn type="submit" id="login-button" color="rgb(247, 52, 52)" style="color: white;">Login
+              </v-btn>
               <v-spacer></v-spacer>
-              <v-btn type="submit">Login</v-btn>
-            </v-card-actions>
+              <v-switch
+                color="rgb(252,4,4)"
+                v-model="switch1"
+                inset
+                :label="`Remember Me`">
+              </v-switch>
+            </v-card-actions>   
           </v-form>
         </v-card>
+            <div id="spacer">
+                <NuxtLink to="/password-reset" style="color: white">
+                Forgot Password?
+                </NuxtLink>
+            </div>
       </v-flex>
     </v-layout>
   </v-container>
 </template>
 
 <script>
 export default {
+  
   auth: false,
   data() {
     return {
-      errors: {},
+      errors: {
+        userName:"please enter a valid user name",
+        password:"please enter a valid password"
+      },
```

Comment:
You can keep this `errors` object as an empty object. Our backend will populate this if something goes wrong. We'll get to that later but for now just assign it to an empty object

```suggestion
      errors: {},
```